django/contrib/postgres/fields/uuid.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib.postgres.functions import RandomUUID
+from django.db.models import UUIDField
+from django.db.models.fields import NOT_PROVIDED, AutoFieldMixin
+
+
+class UUID4Field(UUIDField):
+    def __init__(self, *args, **kwargs):
+        kwargs["db_default"] = RandomUUID()
+        kwargs["default"] = NOT_PROVIDED
+        super().__init__(*args, **kwargs)
+
```

Comment:
@ngnpope [mentioned](https://code.djangoproject.com/ticket/32577#comment:11) that 

```
    def deconstruct(self):
        name, path, args, kwargs = super().deconstruct()
        del kwargs['db_default']
        return name, path, args, kwargs
``` 

would also be needed, but I'm not sure. Can you advise?

Reply:
https://github.com/django/django/pull/18365/commits/68421fb006b286c19f5e3e9fc3e0bca1018f2c96